foosball/games/models.py

```diff
@@ -62,3 +65,94 @@ def __str__(self):
             self.score,
             ", ".join(map(str, self.players.all()))
         )
+
+
+class Player(models.Model):
```

Comment:
Since the `User` model is `User(AbstractUser)` we could also (I think) have these fields on that model. Would that make sense? Of course it might be better to store this information in a separate model like this.